src/app/app.component.ts

```diff
@@ -27,13 +27,57 @@ export class AppComponent implements AfterViewInit {
 
     createChart() {
         this.chartConfig = {
```

Comment:
I believe we need to remove this from app component to within Chart component or may be a separate chart-config,ts
This makes app component clean from any thing other than referring other files.

Reply:
This has been fixed